README.md

```diff
@@ -52,98 +69,54 @@ $ docker run -p 6379:6379 redis
 
-### Authentication
+```
+cp .envrc.example .envrc
+```
+Edit the variables on .envrc adding the real data.
 
-The API we expose for Woocommerce to send its webhook requests requires token
-authentication.
+#### Django
 
-To generate one you need to create a Django superuser and then to create a token
-for it. You can do so running the commands:
+Create a superuser to interact with django
 
 ```
-python manage.py createsuperuser --username pau --email pau@example.com
-python manage.py drf_create_token pau
+python manage.py createsuperuser --username woocommerce --email woocommerce@example.com
 ```
 
-This will give you the token to use with the `Authorization` HTTP header in your
-requests like `Authorization: Token 9944b09199c62bcf9418ad846dd0e4bbdfc6ee4b`.
-See:
-https://www.django-rest-framework.org/api-guide/authentication/#tokenauthentication.
-
-### Running in development
-
-We recommend [HTTPie](https://httpie.io/) to send requests to the API. It helps
-a great deal in managing the request bodies and authentication against the API.
-
-Once installed, make sure you have a user and a token for it in the DB.
-You can build an anonymous session like the one below.
-
-```json
-# session.json
-{
-    "__meta__": {
-        "about": "HTTPie session file",
-        "help": "https://httpie.org/doc#sessions",
-        "httpie": "1.0.3"
-    },
-    "auth": {
-        "password": null,
-        "type": null,
-        "username": null
-    },
-    "cookies": {},
-    "headers": {
-        "Accept": "application/json, */*",
-        "Authorization": "Token <put_your_token_here>"
-    }
-}
+
+Once all previous points are setup run migration to create the necessary tables on django
+
 ```
+python manage.py makemigration
+python manage.py migrate
 
-then you need a dummy JSON body to send:
-
-```json
-# data.json
-{
-  "order_key": "abc",
-  "shipping": {
-    "first_name": "John",
-    "last_name": "Lennon",
-    "address_1": "Beatles Street 66",
-    "address_2": "",
-    "postcode": "08032",
-    "city": "Barcelona",
-    "state": "Barcelona",
-    "country": "ES"
-  },
-  "billing": {
-    "phone": "666666666",
-    "email": "lennon@example.com"
-  },
-  "customer_note": "prueba"
-}
 ```
 
-Now, provided that we point to the Koiki's staging environment by default (see
-`.envrc`), you can run the dev server as `python manage.py runserver` and send
-requests as follows.
+## Usage
 
-```sh
-$ http --json post http://127.0.0.1:8000/api/deliveries/ \
-   --session=./session.json \
-   < data.json
-
-HTTP/1.1 201 Created
-Allow: POST, OPTIONS
-Content-Length: 279
-Content-Type: application/json
-(...)
-```
+### Authentication
+
+The API we expose for Woocommerce to send its webhook uses a signature mechanism. The signature is verified using SignatureValidaton class found [here](https://github.com/coopdevs/lazona_connector/blob/main/api/authentication.py).
```

Comment:
The API we expose for Woocommerce to send its webhook uses a signature mechanism. The signature is verified using our custom [SignatureValidaton](https://github.com/coopdevs/lazona_connector/blob/main/api/authentication.py).

Reply:
Fixed by https://github.com/coopdevs/lazona_connector/pull/92/commits/5bed23c14b42ccace4428b4d99c3b6e32461db34